### PR TITLE
GSP184: p2 to p3 conversion

### PR DESCRIPTION
**Pull request by swetasingh-tudip**
Migrated files to work in Python3.

### CODE REVIEW

File: courses/developingapps/python/datastore/start/quiz/webapp/routes.py [hunk 1]
```diff
@@ -19,14 +19,14 @@
 from flask import send_from_directory
 from flask import request, redirect
 
-import questions
+from . import questions
```
Comment (mylenebiddle) [inline courses/developingapps/python/datastore/start/quiz/webapp/routes.py]: Is it possible to do this with a relative import (without the .)?

Comment (swetasingh-tudip) [reply courses/developingapps/python/datastore/start/quiz/webapp/routes.py]: Without relative import we are getting `ModuleNotFoundError: No module named`  error for `api` and `questions` imports. So we resolve it using explicit relative import in the Python script.

However, we can also use absolute import as below.

```
from quiz.api import api
from quiz.webapp import questions
```

Please, let me know your thoughts on this.

Comment (mylenebiddle) [reply courses/developingapps/python/datastore/start/quiz/webapp/routes.py]: let's use the absolute import referenced in you comment - thanks Sweta!

Comment (swetasingh-tudip) [reply courses/developingapps/python/datastore/start/quiz/webapp/routes.py]: @mylenebiddle updated the python files with the absolute import.
- Thanks!

**Pull request merged**